### DISCUSSION TITLE
Dedicated Arduino Hardware controller

### DISCUSSION THREAD

**Discussion by exoper7** [Rem0o/FanControl.Releases#641] [Ideas]
I have an idea of making a dedicated Arduino-based hardware fan controler for some time but have no idea for GUI. 
Since FanControl is a great GUI and will work with anything (through plugins) there finnaly is motivation to make a hardware. 

I will make something for 6 3Pin/4Pin fans with RPM feedback.

Any of you is interested in custom hardware or have some ideas to share? 
I plan to publish it as Open Hardware.

**Comment by vimaxus** [comment]
I am interested in buying or even trying to build it. Unfortunately not many ideas to share.
